docs/source/reference/kernel.rst

```diff
@@ -26,3 +26,13 @@ JIT kernel definition
    cupyx.jit.syncthreads
    cupyx.jit.shared_memory
    cupyx.jit._interface._JitRawKernel
+
+Kernel binary memoization
+-------------------------
+
+.. autosummary::
+   :toctree: generated/
+   :nosignatures:
+
+   cupy.memoize
+   cupy.clear_memo
```

Comment:
This could also go to "CuPy-specific Functions" I suppose?

Reply:
Yeah, in that sense everything listed in `Custom kernels` page is CuPy-specific. I moved `memorize` to here because it is mainly expected to be used for kernel caching.
Maybe we should consider rewording the "CuPy-specific Functions" heading...